### DISCUSSION TITLE
How to Hide this part???

### DISCUSSION THREAD

**Discussion by Xenavex** [shedaniel/RoughlyEnoughItems#1322] [Q & A] [answered]
![image](https://user-images.githubusercontent.com/84068577/212460184-3b12d6c6-4513-483b-a95d-deddc60912cd.png)
How do i hide this??

**Comment by Xenavex** [comment]
it permanently stays on my inv screen
![image](https://user-images.githubusercontent.com/84068577/212460249-92751727-1717-46ef-b16d-9202bb986007.png)
How do I hide the crafting thing on the top right and how do I hide the + button near my Hotbar
Pls help me with the settings

**Comment by shedaniel** [comment, marked as answer]
Press A on it

**Reply by Xenavex** [reply]
how do i hide the settings and also the 
![image](https://user-images.githubusercontent.com/84068577/214224972-46cfaa68-b19f-4cda-a806-a4241a6a5f09.png)
2 buttons
